diff --git a/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/plugins/load-manifest.js b/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/plugins/load-manifest.js
index b0616a9..f6a3eb0 100644
--- a/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/plugins/load-manifest.js
+++ b/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/plugins/load-manifest.js
@@ -31,7 +31,7 @@ async function getLoadManifestRule(buildOpts) {
     const { outputDir } = buildOpts;
     const baseDir = join(outputDir, "server-functions/default", getPackagePath(buildOpts));
     const dotNextDir = join(baseDir, ".next");
-    const manifests = await glob(join(dotNextDir, "**/{*-manifest,required-server-files,prefetch-hints}.json"), {
+    const manifests = await glob(join(dotNextDir, "**/{*-manifest,required-server-files,prefetch-hints,preview-props}.json"), {
         windowsPathsNoEscape: true,
     });
     const returnManifests = (await Promise.all(manifests.map(async (manifest) => `
@@ -65,7 +65,8 @@ function loadManifest($PATH, $$$ARGS) {
         p.endsWith("server-reference-manifest") ||
         p.endsWith("dynamic-css-manifest") ||
         p.endsWith("fallback-build-manifest") ||
-        p.endsWith("prefetch-hints")) {
+        p.endsWith("prefetch-hints") ||
+        p.endsWith("preview-props")) {
       return {};
     }
   }
